docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -151,6 +151,50 @@ This prop is available on all fields and pickers.
 You can control the field format spacing using the [formatDensity](/x/react-date-pickers/custom-field/#change-the-format-density) prop.
 :::
 
+#### Supported formats
```

Comment:
Have you considered h3 instead? Then this section would be included in the side nav 🤔 
```suggestion
### Supported formats
```

Reply:
I moved to h3 and renamed the title since it's no longer a child of it's previous parent